synapse/replication/slave/storage/events.py

```diff
@@ -48,10 +48,10 @@
 
 class SlavedEventStore(
     RoomMemberWorkerStore,
-    StreamWorkerStore,
     StateGroupWorkerStore,
     UserErasureWorkerStore,
     BaseSlavedStore,
+    StreamWorkerStore,
```

Comment:
Is `StreamWorkerStore` inherited via `BaseSlavedStore` anyway? (Why did we move it this time instead of remove it?)

Reply:
Let me double check this. I'm all `Store`d out!

Reply:
You're spot on! If you'll forgive me for showing off a graph:

![image](https://user-images.githubusercontent.com/8614563/142477689-065d5828-e341-430c-9c9c-5b093f3271c0.png)

As for move versus delete, I suspect I moved it in one commit to solve an MRO conflict, and then didn't notice I could remove it later on.

Reply:
Maybe! Do you think it is worth changing then?

Reply:
Yep, it's removed in 4686190.